### PR TITLE
[CouchDB] new field for behavioural feedback

### PR DESCRIPTION
**Pull request by ridz1208**

### CODE REVIEW

**Comment by driusan** [comment]
@ridz1208 This can't be merged because it needs to be rebased.

**Comment by driusan** [comment]
I don't think this is going to work. There can be (and usually is!) multiple feedbacks in the feedback_thread_bvl table. The join you've added will expand the results to have multiple rows per session.

**Comment by driusan** [comment]
@gluneau & @ridz1208 to test with data that has multiple feedback entries.

**Comment by driusan** [comment]
@gluneau has this been tested yet?

**Comment by driusan** [comment]
This needs a rebase. It can't be merged.

**Comment by ridz1208** [comment]
@gluneau could you please test this query?

File: tools/CouchDB_Import_Demographics.php [hunk 1]
```diff
@@ -135,7 +167,7 @@ function _generateQuery() {
             $EDCFields = ", c.EDC as EDC";
             $fieldsInQuery .= $EDCFields;
         }
-        $concatQuery = $fieldsInQuery . $tablesToJoin . " WHERE s.Active='Y' AND c.Active='Y' AND c.Entity_type != 'Scanner'";
+        $concatQuery = $fieldsInQuery . $tablesToJoin . " WHERE s.Active='Y' AND c.Active='Y' AND c.Entity_type != 'Scanner'"." GROUP BY s.ID";
```
Comment (xlecours) [inline tools/CouchDB_Import_Demographics.php]: Syntax error or access violation: 1055 Expression #14 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'loris.pc_comment.Value' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by

File: tools/CouchDB_Import_Demographics.php [hunk 2]
```diff
@@ -123,8 +127,36 @@ function _getProject($id) {
...
+                                c.ProjectID, 
+                                c.flagged_caveatemptor as CEF, 
+                                c_o.Description as CEF_reason, 
+                                c.flagged_other as CEF_comment, 
+                                pc_comment.Value as Comment, 
+                                COALESCE(pso.Description,'Active') as Status, 
+                                ps.participant_suboptions as Status_reason, 
+                                ps.reason_specify as Status_comments, 
+                                ps.study_consent as Study_consent, 
+                                COALESCE(ps.study_consent_withdrawal,'0000-00-00') AS Study_consent_withdrawal,
```
Comment (xlecours) [inline tools/CouchDB_Import_Demographics.php]: Is the 0000-00-00 a magic value here?

Comment (ridz1208) [inline tools/CouchDB_Import_Demographics.php]: was already there

File: tools/CouchDB_Import_Demographics.php (hunk 1 above)
Comment (ridz1208) [inline tools/CouchDB_Import_Demographics.php]: I cant seem to replicate this error..

Comment (ridz1208) [inline tools/CouchDB_Import_Demographics.php]: I tried on mysql 5.7, also tried it from the import script... never got this error
@driusan any ideas?

File: tools/CouchDB_Import_Demographics.php (hunk 2 above)
Comment (ridz1208) [inline tools/CouchDB_Import_Demographics.php]: @xlecours it seems to be on purpose here... otherwise the coalesce would be useless.

**Comment by ridz1208** [comment]
@xlecours please retest
@gluneau please retest

**Pull request merged**